spa/src/styles/themes.js

```diff
@@ -59,7 +59,16 @@ export const donationPageBase = merge({}, revEngineTheme, {
     lg: '1100px',
     xl: '1300px'
   },
-  font: { body: "'Montserrat', sans-serif", heading: "'Montserrat', sans-serif" }
+  font: { body: "'Montserrat', sans-serif", heading: "'Montserrat', sans-serif" },
+  colors: {
+    cstm_mainHeader: revEngineTheme.colors.white,
+    cstm_mainBackground: '#faf8f8',
+    cstm_formPanelBackground: revEngineTheme.colors.white,
+    cstm_CTAs: revEngineTheme.colors.primary,
+    cstm_ornaments: '#2b2869'
+    cstm_inputBackground: revEngineTheme.colors.white,
+    cstm_inputBorder: revEngineTheme.colors.black
+  }
```

Comment:
@michael-caktus now that I'm looking at this, I'm wondering is there a case in which other parts of the colors object would need to be accessed, beyond the styles editor?  Should I first copy and then modify the `colors` object?

Reply:
Yes, great catch! Something like: 
```suggestion
  font: { body: "'Montserrat', sans-serif", heading: "'Montserrat', sans-serif" },
  colors: {
    ...revEngineTheme.colors,
    cstm_mainHeader: revEngineTheme.colors.white,
    cstm_mainBackground: '#faf8f8',
    cstm_formPanelBackground: revEngineTheme.colors.white,
    cstm_CTAs: revEngineTheme.colors.primary,
    cstm_ornaments: '#2b2869'
    cstm_inputBackground: revEngineTheme.colors.white,
    cstm_inputBorder: revEngineTheme.colors.black
  }
```

BTW, when we've got pre-commit running in CI, merging code-change recommendations via Github usually results in a failed action.

Reply:
Ok wasn't aware.  I'll add to the code and push.